scripts/linters/js_ts_linter.py

```diff
@@ -1137,7 +1137,8 @@ def _get_trimmed_error_output(eslint_output):
             # string('') which is at the index 1 and message-id from the end.
             if re.search(r'^\d+:\d+', line.lstrip()):
                 # Replacing message-id with an empty string('').
-                line = re.sub(r'(\w+-*)+$', '', line)
+                if not 'Parsing error:' in line:
+                    line = re.sub(r'(\w+-*)+$', '', line)
```

Comment:
Have you tested whether the lint test will fail on parsing error now? (Also, we should raise an exception, right?)

Reply:
Yes, Linter is failing now on parsing errors. We are already raising the exception. Eslint is returning its own message along with exception, we are just fixing that message so that part of the message does not get truncated.

Reply:
The one we use in concurrent_task_utils works here
https://github.com/oppia/oppia/blob/1e5ecae642bc9c84699078bf6b216cca9f68f6e8/scripts/concurrent_task_utils.py#L148

Reply:
But we need to raise `Exception` here so that line 148 of concurrent_task_utils can catch? Am I missing anything?

Reply:
I don't think we need an exception here as long as eslint is throwing this error.

Reply:
@DubeySandeep ping!

Reply:
As per offline discussion, the process running on line 1133 https://github.com/oppia/oppia/blob/94cc2130d156e66dfe0fa1ed4d4b26aa1fc0641a/scripts/linters/js_ts_linter.py#L1133 is returning the Error message and linter will fail during CI run.
![Screenshot from 2020-10-19 11-26-44](https://user-images.githubusercontent.com/29057268/96413435-41084a00-1209-11eb-8d83-09966962f548.png)

Reply:
@DubeySandeep PTAL!